commands/Factorio/rconcmd.js

```diff
@@ -0,0 +1,48 @@
+const { MessageEmbed } = require("discord.js");
+const Command = require("../../base/Command.js");
+const serverJS = require("../../servers")
+const RconConnectionManager = require("../../helpers/rcon")
+
+class Linkme extends Command {
+  constructor(client) {
+    super(client, {
+      name: "rconcmd",
+      description: "Send a RCON command (prefixed with /)",
+      usage: "/<command>",
+      examples: ["{{p}}rconcmd /ban DistroByte"],
```

Comment:
I would prefix the command with slash by default

Reply:
That is done by default. This is just a user guide. Will remove  the `/` from the command description